src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -198,6 +198,7 @@ interfaces:
     required_fields:
       - shelf
       - books
+      - series_uuid
```

Comment:
It isn't consistent to have a field in required_fields but not have it in all flattenings.